### PR TITLE
Demon Prison F3

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -4495,15 +4495,15 @@ QUEST_LV_0200_20150323_004494	The stone that is stuck
...
 QUEST_LV_0200_20150323_004501	{memo X}I will protect him for sure
-QUEST_LV_0200_20150323_004502	Tell him that we should prepare little more
-QUEST_LV_0200_20150323_004503	The key of the night star(3)
-QUEST_LV_0200_20150323_004504	I will come back after unleashing the seal
+QUEST_LV_0200_20150323_004502	$Tell her that we should prepare little more
+QUEST_LV_0200_20150323_004503	$The Key of the Night Star (3)
+QUEST_LV_0200_20150323_004504	$I will come back after unleashing the seal
 QUEST_LV_0200_20150323_004505	{memo X}Tell him that more time is needed for the preparation
-QUEST_LV_0200_20150323_004506	The night star in the night
+QUEST_LV_0200_20150323_004506	$The Night star in the Night
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4506]: star -> Star

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -9230,51 +9230,51 @@ QUEST_LV_0200_20151001_009233	The space is gradually collapsing.{nl}I don't know
...
+QUEST_LV_0200_20151001_009238	$The Demon Lord Nuaele's power is weakening the Prison at an alarming rate.{nl}However, we Kupoles have no way to stop Nuaele.{nl}
+QUEST_LV_0200_20151001_009239	$And Kupole Aldona told me that she couldn't just wait for the Revelator...{nl}Without the time for persuasion, she went after Nuaele.
+QUEST_LV_0200_20151001_009240	$Please stop the demons that are coming here through the gaps.{nl}If you could stop them before they join Nuaele, it will be great help to the safety of Aldona.
+QUEST_LV_0200_20151001_009241	$Goddess.{nl}I hope nothing occurs to Aldona...
+QUEST_LV_0200_20151001_009242	$With her forces diminished, Nuaele's power should be weakened.{nl}Before anything happens to Aldona, I will tell you what to do next.
+QUEST_LV_0200_20151001_009243	$Nuaele is difficult to face head-on with only us.{nl}But Nuaele wasn't always that strong.{nl}
+QUEST_LV_0200_20151001_009244	$Nuaele got strong only after obtaining pieces of Hauberk's marks. {nl}Hauberk's subordinates who lost their master started to follow her since she had the pieces of the mark.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:9240]: Why are you renaming Hauberk's seal to Hauberk's mark? Reading off from the Korean, it says 봉인 (seal).

Comment (vtange) [inline QUEST_LV_0200.tsv:9240]: that means the item you collect has an incorrect name. I was matching the dialogue with the item name

Comment (Sunaries) [inline QUEST_LV_0200.tsv:9240]: I haven't done Demon Prison yet, but that is the item name. Maybe @imcgames can shed some light on this.

**Comment by vtange** [comment]
should be good to merge

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**